docs/source/format/StatisticsSchema.rst

```diff
@@ -0,0 +1,885 @@
+
+If module A can pass the statistics associated with the Apache Parquet
+file to module B, module B can use the statistics to optimize its
+query plan.
+
+For example, DuckDB uses this approach but DuckDB couldn't use
+statistics because there wasn't a standardized way to represent
+statistics for the Apache Arrow data.
+
+.. seealso::
+
+   `duckdb::ArrowTableFunction::ArrowScanBind() in DuckDB 1.1.3
+   <https://github.com/duckdb/duckdb/blob/v1.1.3/src/function/table/arrow.cpp#L373-L403>`_
+
+Goals
+-----
+
+* Establish a standard way to represent statistics as an Apache Arrow
+  array.
+
+Non-goals
+---------
+
+* Establish a standard way to pass an Apache Arrow array that
+  represents statistics.
+* Establish a standard way to embed statistics into an Apache Arrow
+  array itself.
+
+Schema
+======
+
+This specification provides only the schema for statistics. This is
+the canonical schema to represent statistics about an Apache Arrow
+dataset as Apache Arrow data.
+
+Here is the outline of the schema for statistics::
+
+    struct<
+      column: int32,
+      statistics: map<
+        key: dictionary<values=utf8, indices=int32>,
+        items: dense_union<...all needed types...>
+      >
+    >
+
+Here is the details of top-level ``struct``:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Data type
+     - Nullable
+     - Notes
+   * - ``column``
+     - ``int32``
+     - ``true``
+     - The zero-based column index, or null if the statistics
+       describe the whole table or record batch.
+
+       The column index is computed as the same rule used by
+       :ref:`ipc-recordbatch-message`.
+   * - ``statistics``
+     - ``map``
+     - ``false``
+     - Statistics for the target column, table or record batch. See
+       the separate table below for details.
+
+Here is the details of the ``map`` of the ``statistics``:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Key or items
+     - Data type
+     - Nullable
+     - Notes
+   * - key
+     - ``dictionary<values=utf8, indices=int32>``
+     - ``false``
+     - The string key is the name of the statistic. Dictionary-encoding is used for
+       efficiency as the same statistic may be repeated for different columns.
+       Different keys are assigned for exact and
+       approximate statistic values. Each statistic has their own description below.
+   * - items
+     - ``dense_union``
+     - ``false``
+     - Statistics value is dense union. It has at least all needed
+       types based on statistics kinds in the keys. For example, you
+       need at least ``int64`` and ``float64`` types when you have a
+       ``int64`` distinct count statistic and a ``float64`` average
+       byte width statistic. See the description of each statistic below.
+
+       We don't standardize field names for the dense union because we
+       can access to proper field by type code not name. So we can use
+       any valid name for fields.
```

Comment:
```suggestion
       Dense union has name for each field but we don't standardize field names for the dense union because we
       can access to proper field by type code not name. So we can use
       any valid name for fields.
```